diagnostic.md

```diff
@@ -48,11 +52,32 @@ Record your responses inside the fenced code blocks below each question.
     value `'123'` inside a Route?
 
     ```md
-    <!-- your response here -->
+    in the model, you have to pass [params.movie_id -1]
```

Comment:
The -1 allows us to access the object with that ID by index, but to get the movie_id itself, simply params.movie_id works. 1/2 credit!